sites/public/pages/applications/contact/address.tsx

```diff
@@ -287,7 +287,11 @@ const ApplicationAddress = () => {
 
             <div className="form-card__group border-b">
               <fieldset>
-                <legend className="field-label--caps">{t("application.contact.address")}</legend>
+                <legend
+                  className={`field-label--caps ${errors.applicant?.address ? "text-red-700" : ""}`}
```

Comment:
Let's try using `text-alert` so it can pick up the `--bloom-color-alert` design token. (I know a lot of other places still use `red-700` for errors but hopefully we can continue to refactor so we aren't hard-coding colors.)